src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -70,31 +90,46 @@ class MoveModal extends React.Component {
     }
   }
 
-  cancelMove = async entries => {
-    const { t } = this.context
-
+  cancelMove = async (entries, trashedFiles) => {
+    const { client } = this.props
     try {
       await Promise.all(
-        entries.map(entry => this.moveEntry(entry._id, entry.dir_id))
+        entries.map(entry =>
+          this.registerCancelable(
+            CozyFile.move(entry._id, { folderId: entry.dir_id })
+          )
+        )
+      )
+      const fileCollection = client.collection(CozyFile.doctype)
+      let restoreErrorsCount = 0
+      await Promise.all(
+        trashedFiles.map(id => {
+          try {
+            this.registerCancelable(fileCollection.restore(id))
+          } catch (e) {
```

Comment:
we will have lint error.

Reply:
Why? (No lint error locally or on Travis)

Reply:
:o ? We don't have "no-used-var" rule ? 

I'm sure we're adding a new more line in our drive console :D